battlecraft-main/src/main/java/br/com/battlebits/battlecraft/warp/Warp1v1.java

```diff
@@ -226,15 +237,28 @@ private boolean hasChallenge(Player target, Player player, ChallengeType type) {
                 && challenges.get(target).get(type).containsKey(player);
     }
 
+    private int tickCount;
+
     @EventHandler
     public void onTick(UpdateEvent event) {
+        if(event.getType() == UpdateEvent.UpdateType.TICK &&  tickCount++ % 7 == 0) {
+            getScoreboard().updateTitleText();
+            for (Player player : getPlayers()) {
+                getScoreboard().updateTitle(player);
+            }
+        }
```

Comment:
Outro caso de repetição de código, se não estiver planejado outro método de titulo, pode ser movido para dentro da Warp em si.

Se for ou não realizado a alteração do local, sugiro o uso de uma constante para o número utilizado para realizar o módulo com o Tick, para facilitar em uma alteração na "velocidade do scroll" futura.

Reply:
Feito!